Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml

```diff
@@ -255,3 +255,28 @@
   - type: ExplosionResistance
     damageCoefficient: 0.5
   - type: GroupExamine
+
+- type: entity
+  parent: ClothingOuterBaseLarge
+  id: ClothingOuterArmorBone
+  name: bone armor
+  description: Seats on you like a second skelet.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Armor/bone_armor.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Armor/bone_armor.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.6
+        Slash: 0.8
+        Piercing: 0.2
```

Comment:
This seems extremely resistent to bullets?

Reply:
well, its bones.